### ISSUE TITLE
The <output> HTML element

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1758]
[output: HTML's native live region element](https://www.scottohara.me/blog/2019/07/10/the-output-element.html)

We should use it for our form notifications, and maybe also for the payday log.

**Comment by rkingett** [comment]
I agree. Native HTML is the best way to go in these cases. It would give a lot of accessibility for free.
